app/main/main-controller.coffee

```diff
@@ -0,0 +1,77 @@
+'use strict'
+
+###*
+ # @ngdoc object
+ # @name main.controller:MainCtrl
+
+ # @description
+
+###
+class MainCtrl
+  constructor: ->
+    @ctrlName = 'MainCtrl'
+
+angular
+  .module('main')
+  .controller 'MainCtrl', [
+    '$scope'
```

Comment:
You already defined controllerAs, so there is no need to use $scope:

http://www.infragistics.com/community/blogs/dhananjay_kumar/archive/2015/07/02/exploring-angular-s-controller-as-syntax-and-the-vm-variable.aspx